lib/transbank/common/request_service.ts

```diff
@@ -20,20 +20,20 @@ const getHeaders = (
 const RequestService = {
   perform: async (request: RequestBase, options: Options) => {
     let requestHeaders = getHeaders(options);
-    return axios({
+    await axios({
       method: request.method,
       url: options.environment + request.endpoint,
       headers: requestHeaders,
       timeout: 10000,
       data: request.toJson(),
     })
-      .then((response) => {
+      .then((response: any) => {
         if (response.status == 204) {
           return true;
         }
         return response.data;
```

Comment:
It fails in this line, it shouldn't be a race condition if it gets to this point. If it were a problem with the response it should fail in line 31

Reply:
The error says .data of undefined doesn't exist